src/Resources/public/js/property/config-form.tsx

```diff
@@ -20,6 +20,8 @@ export type ConfigValuesType = {
 };
 type StateType = {
     configValues: ConfigValuesType;
+    initialConfig: ConfigValuesType;
+    isSaved: boolean;
```

Comment:
I would consider renaming `isSaved` to `isSaving` or something similar.
`isSaved` is used more like a "is currently saving" instead of a "has data been saved" manner, therefore I think the current name is a bit misleading/confusing.